dist/js/example/property/base.json

```diff
@@ -9,7 +9,8 @@
     "data": {
         "name": "pressure",
         "units": "kbar",
-        "value": -149.03
+        "value": -149.03,
```

Comment:
the name of the file could be `holder.json` instead of base

Reply:
Done